addon/components/o-s-s/badge.ts

```diff
@@ -40,10 +51,30 @@ export default class OSSBadge extends Component<OSSBadgeArgs> {
     return SizeDefinition[size as SizeType];
   }
 
+  get skinClass(): string {
+    const skin: SkinType = this.args.skin;
+
+    if (!skin) return '';
+
+    if (!Object.keys(SkinDefinition).includes(skin as SkinType)) {
+      throw new Error(
+        `[component][OSS::Badge] Unknown skin. Available skins are: ${Object.keys(SkinDefinition).join(', ')}`
+      );
+    }
+
+    return SkinDefinition[skin as SkinType];
+  }
+
   get computedClass(): string {
     const classes = ['upf-badge', 'upf-badge--shape-round'];
 
-    classes.push(this.sizeClass);
+    [this.sizeClass, this.skinClass].forEach((klass) => {
+      classes.push(klass);
+    });
```

Comment:
```suggestion
    [this.sizeClass, this.skinClass].forEach((class) => {
      classes.push(class);
    });
```
`klass` with a `k`? :laughing:

Reply:
Because `class` is a keyword 😅

Reply:
t'as la Klass ou tu l'as pas, c'est tout!